examples/llm-inference/build.envd

```diff
@@ -0,0 +1,18 @@
+# syntax=v1
+def build():
+    # base(os="ubuntu20.04", language="python3")
+    base(dev=True)
+    install.conda()
+    install.python()
+    install.cuda(version="11.2.2", cudnn="8")
+    install.python_packages(
+        [
+            "torch",
+            "torchvision",
+            "bitsandbytes",
+            "accelerate",
+            "--extra-index-url https://download.pytorch.org/whl/cu113",
+        ]
+    )
+
+    install.python_packages(["git+https://github.com/huggingface/transformers.git"])
```

Comment:
Why should we install the transformers via git?

Reply:
Tested. We can install it from PyPI directly. No need to use the latest code.

Reply:
Then could we remove it?